packages/cursorless-engine/src/cursorlessEngine.ts

```diff
@@ -65,18 +66,20 @@ export function createCursorlessEngine(
 
   ide.disposeOnExit(rangeUpdater, languageDefinitions, hatTokenMap, debug);
 
+  const commandRunnerDecorators = new Array<CommandRunnerDecorator>();
```

Comment:
Nitpicking here, but we tend to prefer
```suggestion
  const commandRunnerDecorators: CommandRunnerDecorator[] = [];
```